Yosemite/YosemiteTests/Mockups/Networking/Remote/MockShippingLabelRemote.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import Networking
+
+import XCTest
+
+/// Mock for `ShippingLabelRemote`.
+///
+final class MockShippingLabelRemote {
+    private struct PrintResultKey: Hashable {
+        let siteID: Int64
+        let shippingLabelID: Int64
+        let paperSize: String
+    }
+
+    /// The results to return based on the given arguments in `loadProduct`
```

Comment:
nit: I think this is a typo -> `loadProduct`

Reply:
fixed in: https://github.com/woocommerce/woocommerce-ios/pull/3156/commits/88bc08df311eeb0c06fa17fc8f9e9c08599f2b9a